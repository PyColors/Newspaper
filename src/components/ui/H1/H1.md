      <H1>coco</H1>