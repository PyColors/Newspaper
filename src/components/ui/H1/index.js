import styled from 'styled-components';
import theme from '../../../styles/theme';

const { colors, fontSize } = theme;

const H1 = styled.h1`
  font-size: ${fontSize.xlarge};
  color: ${props => props.color || `${colors.primaryTwo}`};
`;

export default H1;
