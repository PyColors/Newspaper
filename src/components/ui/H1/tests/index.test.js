import React from 'react';
import { shallow } from 'enzyme';

import H1 from '../index';

describe('<H1 />', () => {
  it('should render an <H1> tag', () => {
    const renderedComponent = shallow(<H1 />);
    expect(renderedComponent.type()).toEqual('h1');
  });

  it('should render its text', () => {
    const children = 'Text';
    const renderedComponent = shallow(<H1>{children}</H1>);
    expect(renderedComponent.contains(children)).toBe(true);
  });

  it('should render a prop', () => {
    const id = 'testH1Id';
    const renderedComponent = shallow(<H1 id={id} />);
    expect(renderedComponent.prop('id')).toEqual(id);
  });
});
