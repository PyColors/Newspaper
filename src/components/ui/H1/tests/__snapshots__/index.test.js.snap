// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<H1 /> should match the snapshot 1`] = `
<h1
  class="className"
  className="sc-bdVaJa jvORyl"
>
   
  <span>
    Description
  </span>
  &gt;
</h1>
`;
